New Medley primer

pamoroso opened the pull request:
This PR changes the [documentation page](https://interlisp.org/software/using-medley) to link to the new Medley primer [Medley Interlisp for the Newcomer](https://primer.interlisp.org), mentions it on the [home page](https://interlisp.org), and announces the primer on the [News and Status Reports](https://interlisp.org/project/status) page.

pamoroso committed:
Encourage to check out the new Medley primer

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso committed:
Add new Medley primer entry

Rewrite the entry for the Medley primer to link to the new primer and deprecate the old one.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso committed:
Add new Medley primer entry

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

stumbo commented on content/en/project/status/_index.md:
```diff
@@ -14,6 +14,10 @@ aliases:
 
+## New Medley Primer
+
+In november of 2025 we published [Medley Interlisp for the Newcomer](https://primer.interlisp.org) by Abhik Hasnain, an introductory guide to Interlisp and the Medley environment.
```
Capitalize the N in November.

stumbo commented on content/en/software/using-medley/_index.md:
```diff
@@ -34,7 +34,7 @@ Note: locations of documents are likely to change. Best to bookmark this page, w
 If you are familiar with Common Lisp, this guide points out some differences.
 3. [Using Medley Online](online-using) -- additional introduction primarily for online users
-1. [Medley for the Novice](/documentation/Medley-Primer.pdf) (also known as Medley Primer).  An introductory guide to the basics of Medley such as executing commands, using menus and files, manipulating windows, editing and saving Lisp code, using the development tools, and more. Read it in full. The code in chapter 20 "Free Menus" doesn't work and some illustrations are missing.
+1. [Medley Interlisp for the Newcomer](https://primer.interlisp.org) (also known as the new Medley Primer). An introductory guide to the basics of Interlisp and the Medley environment that doesn't assume prior knowledge of Lisp. Read it in full. Written in 2025, the guide is designed to ease modern users into the system and replace the [old primer](/documentation/Medley-Primer.pdf) published in 1 ...
```
I would end the last sentence at 'now obsolete.'  

Also, is it worth reordering this list.  And moving this item to the top of the introductory material?

pamoroso replied:
I rearranged section *Introductory material* to move the new primer entry to the beginning of the list, delete the reference to the errors of the old primer, and remove material the new primer now covers.

pamoroso committed:
Update section Introductory material

Remove entries Interlisp Basics and Using Medley Online as the new Medley primer now covers same material. Move the new primer entry to the beginning of the list and remove the irrelevant reference to errors.

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>

pamoroso committed:
Capitalize November

Signed-off-by: Paolo Amoroso <info@paoloamoroso.com>
